javascript/react-verbs/src/verbs/shared/FilterFunction/FilterFunction.tsx

```diff
@@ -126,17 +147,22 @@ export const FilterFunction: React.FC<FilterFunctionProps> = memo(
 			<Container>
 				<SideBySide>
 					{operatorDropdown}
-					<ColumnOrValueComboBox
-						required={!suppressLabels}
-						table={table}
-						filter={columnFilter}
-						disabled={isEmpty}
-						label={suppressLabels ? undefined : 'Comparison value'}
-						placeholder={placeholder}
-						text={criterion.value ? `${criterion.value}` : undefined}
-						onChange={handleComboBoxChange}
-						styles={narrowDropdownStyles}
-					/>
+					{type === DataType.Date ? (
```

Comment:
This one is kind of tricky. If we just show the calendar for dates, we lose the ability to compare to another column